posix.mak

```diff
@@ -61,14 +61,7 @@ dustmite:  $(ROOT)/dustmite
 $(ROOT)/dustmite: DustMite/dustmite.d DustMite/splitter.d
 	$(DMD) $(DFLAGS) DustMite/dustmite.d DustMite/splitter.d -of$(@)
 
-#dreadful custom step because of libcurl dmd linking problem (Bugzilla 7044)
```

Comment:
It's quite interesting that this isn't needed anymore, but `pragma(lib, "curl")` does the trick now.